README.md

```diff
@@ -118,20 +118,20 @@ Contributors are welcomed to join the FEATS project. Please check [CONTRIBUTING]
 <img src="https://upload-images.jianshu.io/upload_images/4420767-4e95b186a1a1bfba.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240"  height="300" width="300">
 
 
-## Fescar ecosystem
+## Seata ecosystem
 
-* [Fescar Ecosystem Entry](https://github.com/fescar-group) - A GitHub group `fescar-group` to gather all Fescar relevant projects not appropriate in [alibaba](https://github.com/alibaba) group yet
-* [Fescar Samples](https://github.com/fescar-group/fescar-samples) - Samples for Fescar
-* [Fescar Docker](https://github.com/fescar-group/fescar-docker) - Fescar integration with docker
-* [Fescar K8s](https://github.com/fescar-group/fescar-k8s) - Fescar integration with k8s
-* [Awesome Fescar](https://github.com/fescar-group/awesome-fescar) - Description of Fescar related projects 
-* [Fescar Website](https://github.com/fescar-group/fescar.io) - Fescar official website （***In the process of design***）
+* [Seata Ecosystem Entry](https://github.com/fescar-group) - A GitHub group `fescar-group` to gather all Fescar relevant projects not appropriate in [alibaba](https://github.com/alibaba) group yet
```

Comment:
A GitHub group `seata-group`,....all Seata relevant

Reply:
When to transfer the projects under https://github.com/fescar-group/awesome-fescar